library/Icinga/Rememberme/Common/Database.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Icinga\Rememberme\Common;
+
+use Icinga\Application\Config as IcingaConfig;
+use Icinga\Data\ResourceFactory;
+use ipl\Sql\Config as SqlConfig;
+use ipl\Sql\Connection;
+use PDO;
+
+trait Database
+{
+    protected function getDb()
+    {
+        $config = new SqlConfig(ResourceFactory::getResourceConfig(
+            IcingaConfig::module('customer')->get('backend', 'resource')
```

Comment:
Let's just hardcode the resource name to `remember-me` for the moment.
When we've implemented everything, we merge the schema into Icinga Web's schema and adapt the trait accordingly.